evap/results/views.py

```diff
@@ -50,7 +53,7 @@ def _delete_course_template_cache_impl(course):
 
 def warm_up_template_cache(evaluations):
     evaluations = get_evaluations_with_course_result_attributes(get_evaluations_with_prefetched_data(evaluations))
-    current_language = translation.get_language()
+    current_language = get_language()
```

Comment:
What's the reason for this change? We need one additional import this way, right?

Reply:
I also used `get_language` down in `evaluation_text_answers_export`:
```python
filename = "Evaluation-Text-Answers-{}-{}-{}.xls".format(evaluation.course.semester.short_name, evaluation.full_name, get_language())
```
to make the line shorter. Should I just expand both to `translation.get_language()` or leave it as is?

Reply:
Ah okay. I'd leave the import as it was and use `translation.get_language()`. The line length could be reduced by just adding some newlines
```python
filename = "Evaluation-Text-Answers-{}-{}-{}.xls".format(
    evaluation.course.semester.short_name,
    evaluation.full_name,
    translation.get_language(),
)
```

What do you think about that?

Reply:
Fair enough